CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2024-10-25
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled as `v20241025`.
+
+### Announcements
+
+* AKS version 1.30 is now available as a [Long term support version](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#aks-kubernetes-release-calendar) and AKS version 1.28 End of Life is Jan, 15 2025.
+* Upgrade from LTS 1.27 to LTS 1.30 is now supported. 
```

Comment:
These were present in the previous release note too. Maybe we can remove the second one. For first one, should we just keep the 1.28 EOL part?

Reply:
Lets move 1.28 EOL to its own line item and please put EOL date as Jan 31 2025 (likely to be end of Jan).